Source/WebCore/css/parser/CSSSelectorParserContext.cpp

```diff
@@ -42,6 +42,7 @@ CSSSelectorParserContext::CSSSelectorParserContext(const CSSParserContext& conte
     , highlightAPIEnabled(context.highlightAPIEnabled)
     , popoverAttributeEnabled(context.popoverAttributeEnabled)
     , thumbAndTrackPseudoElementsEnabled(context.thumbAndTrackPseudoElementsEnabled)
+    , imageControlsEnabled(context.imageControlsEnabled)
     , viewTransitionsEnabled(context.propertySettings.viewTransitionsEnabled)
```

Comment:
Why does this code have viewTransitionsEnabled but the equivalent code in CSSParserContext.cpp does not?

Reply:
`viewTransitionsEnabled` is on `propertySettings` in `CSSParserContext` which is auto-generated by CSSProperties.json